src/apps/inventory/templates/product_list.html

```diff
@@ -1,57 +1,23 @@
 {% extends 'base.html' %}
 
 {% block content %}
+  {% block meta %}
+    <title>Product List</title>
+  {% endblock meta %}
+
   <div class="container mt-4">
     <h1>Product List</h1>
+  </div>
 
-    <!-- Filter Form Card -->
-    <div class="card mb-4">
-      <div class="card-body">
-        <form method="get">
-          {{ filter.form.as_p }}
-          <button type="submit" class="btn btn-primary">Apply Filters</button>
-        </form>
-      </div>
-    </div>
-
-    <!-- Ordering Form Card -->
-    <div class="card mb-4">
-      <div class="card-body">
-        <form method="get" class="ordering-form">
-          <div class="form-group">
-            <label for="order_by">Order by:</label>
-            <select id="order_by" name="order_by" class="form-control">
-              <option value="">None</option>
-              <option value="name">Name (A-Z)</option>
-              <option value="-name">Name (Z-A)</option>
-              <option value="price">Price (Low to High)</option>
-              <option value="-price">Price (High to Low)</option>
-            </select>
-          </div>
-          <button type="submit" class="btn btn-primary">Apply Order</button>
-        </form>
-      </div>
-    </div>
+    <!-- Filters-->
+  {% include 'utils/filters.html' %}
```

Comment:
can we create {% block %} for filters? Lets include it once in product base template
hm?

Reply:
once we create product base template we will inherit our templates from it.
it will take 1 line if code, the same we have here. but we can see for sure on which pages we use filter block and on which we don't. so, i think it's better to keep it like this.

Reply:
all pages with product views have filters

maybe we can extend base template and create base_list view or smth 
where we will have sort options and filter options by default?